WeakAuras/Prototypes.lua

```diff
@@ -2195,8 +2200,6 @@ Private.event_prototypes = {
           end
         end,
         test = "unitisunit",
-        store = true,
-        conditionType = "bool",
```

Comment:
This condition did nothing and was particular useless. I looked at whether it's fixable, since it does sound useful, but unfortunately that appears to be not easy.